libcxx/docs/VendorDocumentation.rst

```diff
@@ -257,7 +257,7 @@ General purpose options
 
   A semicolon list of arguments to pass when running the libc++ benchmarks using the
   ``check-cxx-benchmarks`` rule. By default we run the benchmarks for a very short amount of time,
-  since the primary use of ``check-cxx-benchmarks`` is to get test and sanitizer coverage, not to
+  since the primary use of ``check-cxx-benchmarks`` is to get test and coverage sanitizer, not to
```

Comment:
This change is wrong. This documentation relates to "coverage for sanitizers", not a specific kind of "sanitizer for coverage".

Reply:
Nice catch! I did another pass on the PR to make sure there wasn't other such instances